crowplexus/hscript/Interp.hx

```diff
@@ -931,8 +967,14 @@ class Interp {
 				default:
 			}
 			return null;
+		}, function(o: Dynamic, ?f: String):Bool {
+			return switch (Type.typeof(o)) {
+				case (TFloat | TInt): (f == 'hex');
+				case TClass(String): (f != 'hex');
+				default: false;
+			}
 		}),
-		new UsingEntry("Lambda", function(o: Dynamic, f: String, args: Array<Dynamic>): Dynamic {
```

Comment:
I had it as a string to allow people make custom usings,

but please do wait until we push our local changes